project/src/main/java/com/google/sps/servlets/EventCreationServlet.java

```diff
@@ -33,34 +34,40 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
 
     // If event DNE, sends 404 ERR to frontend
     if (!eventOptional.isPresent()) {
```

Comment:
Liangqi mentioned in one of my PRs that flipping this logic around would be clearer to read, so the writing should be in the if() and the 404 in the else()